.github/workflows/pr-e2e.yml

```
@@ -93,6 +93,17 @@ jobs:
           gh pr checkout ${{ needs.triage.outputs.pr_num }}
           git checkout ${{ needs.triage.outputs.commit_sha }}
 
+      - name: Run regex checks
+        env:
+          COMMENT_BODY: ${{ github.event.comment.body }}
+        run: |
+          MESSAGE="$COMMENT_BODY"
+          REGEX='/run-e2e (.+)'
+          if [[ "$MESSAGE" =~ $REGEX ]]
+          then
+            export E2E_TEST_REGEX="$(echo ${BASH_REMATCH[1]} | head -1)"
+          fi
+
```

We need to include there the failure of the check and the feedback:
```suggestion

      - name: React to comment with failure
        uses: dkershner6/reaction-action@v2
        if: steps.regex-validation.outcome != 'success'
        with:
          token: ${{ secrets.GITHUB_TOKEN }}
          commentId: ${{ github.event.comment.id }}
          reaction: "-1"

      - name: Set status failure
        uses: LouisBrunner/checks-action@6b626ffbad7cc56fd58627f774b9067e6118af23 # v2
        if: steps.regex-validation.outcome != 'success'
        with:
          token: ${{ secrets.GITHUB_TOKEN }}
          sha: ${{ needs.triage.outputs.commit_sha }}
          name: ${{ env.E2E_CHECK_NAME }}
          conclusion: failure
          details_url: https://github.com/${{github.repository}}/actions/runs/${{github.run_id}}

```